lang/mlton-bootstrap/Portfile

```diff
@@ -116,6 +116,12 @@ post-patch {
 compiler.c_standard 2011
 compiler.blacklist-append {clang < 900}
 
+# mlton-bootstrap bakes in C compiler into its script, like R does.
+# We have to ensure that both mlton-bootstrap and mlton use the identical compiler.
+# For now set it to use clang-16 or gcc-13. When newer compilers are enabled,
+# it should be done for mlton-bootstrap and mlton together, with revbumping both.
+compiler.blacklist-append {macports-clang-1[7-9]} {macports-gcc-1[4-9]}
+
```

Comment:
Why not bake in the latest compiler instead of older ones?

Reply:
These are default ones for C11, AFAIK. Clang-17 only used for C17+. What we are trying to accomplish is to control compiler choice via portfile, like with R. Not allowing the base to pick whatever it wants depending on OS defaults, since those defaults are changed, and then we get a breakage.

Reply:
(Well, we could blacklist older clangs as well and force clang-17, but what problem it gonna solve? Personally I do not really care which clang version is used, I just wanna ensure the port is not broken.)